### ISSUE TITLE
Create "Getting started" guide for Node.js beginners

### ISSUE THREAD

**Issue by hanzgit** [nodejs/nodejs.org#495]
I usually use windows7 and have never use node.js in any platform.
In this time I start to use node.js.

For the first time, I tried to install node.js on windows7
and feel hard to know how to check operation after installation.

I estimate to be described about operation check at
 node.js HP(https://nodejs.org/en/)
  ->[DOWNLOADS](https://nodejs.org/en/download/)
  ->[Installing Node.js via package manager](https://nodejs.org/en/download/package-manager/)
  ->[Windows](https://nodejs.org/en/download/package-manager/#windows)
(but I counldn't find descrption.)  

Through ohter engineer's blog,
I know to be described about operation check at [ABOUT](https://nodejs.org/en/about/)
and feel better to be described at that place([DOWNLOADS]->[Installing Node.js via package manager]->[Windows]).

In addition I feel better to be described about execution command(node filename.js).

****_example**_****

(Add describe Under [DOWNLOADS]->[Installing Node.js via package manager]->[Windows](%28https://nodejs.org/en/download/package-manager/#windows%29))

 Windows

 1.Simply download the Windows Installer directly from the nodejs.org web site.
1. Check operation
   
   2-1. Create example.js.
   
   ==example.js==

``` js
 const http = require('http');

 const hostname = '127.0.0.1';
 const port = 1337;

 http.createServer((req, res) => {
   res.writeHead(200, { 'Content-Type': 'text/plain' });
   res.end('Hello World\n');
 }).listen(port, hostname, () => {
   console.log(`Server running at http://${hostname}:${port}/`);
 });
```

 ================

 2-2. Run command prompt

 2-3. Change directory to directory where example.js exists

 2-4. execution command [node example.js]

 2-5. Run Internet Browser at [http://127.0.0.1:1337/]

****_example**_****

**Comment by kahwee** [comment]
Do you mean that you like instructions on how to validate that your Node.js installation is working correctly?

**Comment by ghost** [comment]
if you want to confirm that your node.js has installed correctly, then the steps you have described should be enough. if that is not your issue, could you give some more information please? we would appreciate it :)

**Comment by benjamingr** [comment]
@hg @kahwee I think OP is suggesting that we add these sort of instructions to the website.

**Comment by wonderdogone** [comment]
I think the ask is for a "getting started" in the guides? https://nodejs.org/en/docs/guides/?

**Comment by fhemberger** [comment]
If anyone wants to help creating a simple "getting started" guide (aka "How do I start with Node.js after I downloaded/installed it"), please feel free to submit a PR.

**Comment by stevemao** [comment]
ping @nodejs/documentation

**Comment by eljefedelrodeodeljefe** [comment]
I regard Kubernetes' getting started guides as really nice. The sheer amount is too bloated for us (they needed though naturally), but they usually serve as a good starting point not only for novices.

https://github.com/kubernetes/kubernetes/tree/master/docs/getting-started-guides

**Comment by Knighton910** [comment]
+1 `getting started guide`

**Comment by fhemberger** [comment]
Fixed by #1308.
